includes/UserMailer.php

```diff
@@ -222,56 +214,46 @@ public static function send( $to, $from, $subject, $body, $replyto = null, $cont
 		$headers['From'] = $from->toString();
 		$headers['Return-Path'] = $from->address;
 
-		if ( $replyto && $replyto instanceof MailAddress /* Wikia change (BugId:6519) */ ) {
+		if ( $replyto && $replyto instanceof MailAddress ) {
 			$headers['Reply-To'] = $replyto->toString();
 		}
 
 		$headers['Date'] = date( 'r' );
 		$headers['MIME-Version'] = '1.0';
 
-		/* <Wikia> */
-		if(empty($attachements)) {
-		/* </Wikia> */
+		if ( empty( $attachements ) ) {
 			$headers['Content-Type'] = ( is_null( $contentType ) ?
 				'text/plain; charset=UTF-8' : $contentType );
 			$headers['Content-Transfer-Encoding'] = '8bit';
-		/* <Wikia> */
 		}
-		/* </Wikia> */
 
 		$headers['Message-ID'] = self::makeMsgId();
 		$headers['X-Mailer'] = 'MediaWiki mailer';
 
-		# <Wikia> merged 1.19 (MoLi)
 		$headers['X-Msg-Category'] = $category;
 		if ( $priority ) {
 			$headers['X-Priority'] = $priority;
 		}
-		# </Wikia>
 
-		$ret = wfRunHooks( 'AlternateUserMailer', array( $headers, $to, $from, $subject, $body
-		/*<Wikia>*/
+		$ret = wfRunHooks( 'AlternateUserMailer', [ $headers, $to, $from, $subject, $body
 			, $priority, $attachements
```

Comment:
As part of cleaning up, might as well put this on the same line as above and eliminate this weird stray leading comma.